zipkin-server/src/test/java/zipkin2/server/internal/elasticsearch/InitialEndpointSupplierTest.java

```diff
@@ -0,0 +1,41 @@
+package zipkin2.server.internal.elasticsearch;
+
+import com.linecorp.armeria.client.Endpoint;
+import com.linecorp.armeria.client.endpoint.DynamicEndpointGroup;
+import java.util.Arrays;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class InitialEndpointSupplierTest {
+  static final Endpoint FOO = Endpoint.of("foo");
+  static final Endpoint BAR = Endpoint.of("bar");
+  static final Endpoint CAT = Endpoint.of("cat");
```

Comment:
do we really want to mix cats and dogs in the same endpoint group?

Reply:
I guess I forgot to add the health check here 🤣